src/core_landice/mode_forward/mpas_li_iceshelf_melt.F

```diff
@@ -179,7 +184,8 @@ subroutine li_basal_melt_floating_ice(domain, err)
       endif
 
       ! Simple options handled here
-      if (trim(config_basal_mass_bal_float) == 'none') then
+      if (trim(config_basal_mass_bal_float) == 'none' &
+                .and. config_front_mass_bal_grounded == 'none') then
```

Comment:
I'm not seeing how the value of `config_front_mass_bal_grounded` should affect this option of `config_basal_mass_bal_float`.  Unless you have an explanation I'm not thinking of, this change should be removed.

Reply:
Removed